autorest/utility.go

```diff
@@ -138,11 +138,31 @@ func MapToValues(m map[string]interface{}) url.Values {
 	return v
 }
 
+// ToStringSlice method converts interface{} to []string.
+func ToStringSlice(s interface{}) []string {
+
+	v := reflect.ValueOf(s)
```

Comment:
Please add a test here to make sure that `v.Kind()` is a `reflect.Slice` or a `reflect.Array`.